### PR TITLE
Fx 516 fix build macros

### PR DESCRIPTION
**Pull request by syegres**

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -409,7 +409,7 @@ AC_ARG_WITH(poco,
   AS_HELP_STRING([--with-poco=DIR],[location of Poco]),
 [
 	CPPFLAGS="$CPPFLAGS -I${withval}/include"
-	LDFLAGS="$LDFLAGS -L${withval}/lib"
+	LDFLAGS="$LDFLAGS -L${withval}OD/lib"
```
Comment (dakka) [inline configure.ac]: Not sure why this change is here - what does OD do? ${withval} is a directory name

Comment (syegres) [inline configure.ac]: Seems a mistake. Please, revert this one

**Comment by dakka** [comment]
Pls resubmit with fix
